### ISSUE TITLE
ES4x install fails due to 'Graal version mismatch' error

### ISSUE THREAD

**Issue by UglyHobbitFeet** [reactiverse/es4x#408]
My build fails during the es4x install step. Even though I have Graal installed and setup, es4x cannot find it and exits with an error. I see there's a version mismatch in graaljs in the es4x versions section below (20.1.0 vs 20.2.0).  I also see es4x versions is showing 0.13.0 whereas @es4x/create is 0.13.1. Could either of these be the issue?

I have downloaded and setup graal v20.2.0 ce for linux as per instructions here: https://www.graalvm.org/docs/getting-started-with-graalvm/linux/

Sanity check:
```
$ which npm
~/Downloads/graalvm-ce-java8-20.2.0/bin/npm

$ which java
~/Downloads/graalvm-ce-java8-20.2.0/bin/java

$ echo $PATH
/home/<myUser>/Downloads/graalvm-ce-java8-20.2.0/bin:otherStuff...

$ echo $JAVA_HOME
/home/<myUser>/Downloads/graalvm-ce-java8-20.2.0

$ echo $CLASSPATH
.:/home/<myUser>/Downloads/graalvm-ce-java8-20.2.0/jre/lib:/home/<myUser>/Downloads/graalvm-ce-java8-20.2.0/lib:/home/<myUser>/Downloads/graalvm-ce-java8-20.2.0/lib/tools.jar

$ which es4x
/bin/es4x

$ es4x versions
VM:      OpenJDK 64-Bit Server VM GraalVM CE 20.2.0 - 1.8.0_262
ES4X:    0.13.0
graaljs: 20.1.0

$ cd myProject
$ rm -rf node_modules
$ rm package-lock.json
$ es4x init
$ npm install
> <builds as normal then comes to this step>...
> es4x install

Runtime GraalVM version mismatch { wanted: [20.2.0], provided: [0.0.0] }
For installation help see: https://www.graalvm.org/docs/getting-started-with-graalvm/
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```
A relevant snippet of my package.json looks like this:
```
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "postinstall": "es4x install",
    "start": "es4x",
   ...
  },
  "dependencies": {
    "@es4x/create": "^0.13.1",
    "@vertx/config": "^3.9.3",
    "@vertx/core": "^3.9.3",
    "@vertx/mongo-client": "^3.9.3",
    "@vertx/web": "^3.9.3",
    "@vertx/web-client": "^3.9.3",
   ...
  },
```

FWIW I have also explicitly called es4x install from the root of my project and no errors occur (instead of npm install calling it). It exits with no output message (success or error).

**Comment by UglyHobbitFeet** [comment]
Also, slight side-note. Do we still need es4x-pm now that @es4x/create exists? I'm unclear of the differences of how/when to use each.

**Comment by UglyHobbitFeet** [comment]
As another sidenote, I found a temporary workaround in the interim.  It was to change the symbolic link for /bin/es4x to point to the @es4x/create instead of the es4x-pm.
``` 
cd <myProject>
npm install @es4x/create
cd /bin
ln -sfn /<myProject>/node_modules/@es4x/create es4x
```

**Comment by pmlopes** [comment]
@UglyHobbitFeet indeed, there's a bug with the graal finder. If you use the java11 alternative for now it should work until the next release.

In the meantime I'll deprecate es4x-pm as it is easier to use @es4x/create.

My future goal is to avoid having a install step (it should be automated for development) only needed manually for docker/jlink

**Comment by UglyHobbitFeet** [comment]
Hi Paulo, thanks for looking into this!  Unfortunately I'm bound to Java8 so I don't know if I can take advantage of that.  I'll have to use my workaround until your next release.

**Comment by pmlopes** [comment]
I'll backport the fix to 0.13.2 and release it asap.

**Comment by pmlopes** [comment]
a new release 0.13.2 should is being pushed to maven central and npm atm.
